package.json

```diff
@@ -29,6 +29,7 @@
     "@instructure/ui-eslint-config": "^8.43.1",
     "@instructure/ui-stylelint-config": "^8.43.1",
     "@instructure/ui-webpack-config": "^8.43.1",
+    "@oxctl/lti-auto-configuration": "^1.1.8",
```

Comment:
Thanks! I released 1.1.8 this morning with the fix to support tools without API keys.